### PR TITLE
Allow adding existing users to rotations

### PR DESCRIPTION
**Pull request by dfaulken**
Addresses #44.

### CODE REVIEW

**Comment by dfaulken** [comment]
screenshot from 2017-04-17 15-47-51

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -22,9 +22,21 @@ def destroy
 
   def index
     @users = @roster.users
+    @other_users = User.all - @users
     @fallback = @roster.fallback_user
   end
 
+  def transfer
```
Comment (Anbranin) [inline app/controllers/users_controller.rb]: 'transfer' makes me think we're moving a user from one roster to another.

Comment (Anbranin) [inline app/controllers/users_controller.rb]: maybe just `add_roster`?

Comment (Anbranin) [inline app/controllers/users_controller.rb]: eh but that's using the word 'roster' more, so I'm not really hooked on changing this.

**Pull request merged**